.ci/docs/crash-debug.md

```
@@ -0,0 +1,77 @@
+```yaml
+DEBUG_MOUNT_PATH: "/enroot_images"
+DEBUG_RETENTION_DAYS: "14"   # informational — no automated cleanup yet
+ENROOT_DATA_PATH: "/enroot-data/enroot-data-148069/user-148069"
+```
+
+`ENROOT_DATA_PATH` must point to the svc-nixl user's enroot data directory on the mizu
+nodes — enroot requires it to locate the container's writable layer.
+
+## File naming convention
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)